e2e/libs/keywords/persistentvolumeclaim_keywords.py

```diff
@@ -20,9 +20,9 @@ def cleanup_persistentvolumeclaims(self):
         for claim in claims.items:
             self.delete_persistentvolumeclaim(claim.metadata.name)
 
-    def create_persistentvolumeclaim(self, name, volume_type="RWO", sc_name="longhorn"):
+    def create_persistentvolumeclaim(self, name, volume_type="RWO", sc_name="longhorn", storage_size="3Gi"):
```

Comment:
By running test cases based on this PR, I got many test cases failed with error: `ValueError: Invalid size format: 3Gi`:

https://ci.longhorn.io/job/private/job/longhorn-e2e-test/2036/console